next/components/forms/simple-components/MenuDropdown/MenuDropdown.tsx

```diff
@@ -7,7 +7,7 @@ export type MenuItemBase = {
   id?: number
   title: string
   icon?: ReactNode
-  onPress?: () => Promise<void> | void
+  onPress?: () => Promise<void> | Promise<boolean> | void
```

Comment:
would rather not return the boolean where that happens and instead make such function return void